ui/scss/init/_gui.scss

```diff
@@ -257,3 +257,10 @@ a {
 .download-text {
   font-size: var(--font-xsmall);
 }
+
+.notice-message {
```

Comment:
We don't have any CSS for a generic message at the top of a screen, it's pretty basic so I just added it here.